.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Short summary of your changes. Explain what have you done and comment the decisions that you may have taken.
+
+### Related Trello cards:
+* Add a link for each Trello cards related to this PR. Don't forget to update the cards status in Trello.
```

Comment:
For me is the same, personally I would delete it after I complete the field during the pull request. So if it's commented or not is the same because I'll delete it anyways. What do you think? Would you leave it like a comment after complete the pull request template?